src/ui/profile_preferences.ui

```diff
@@ -1984,77 +1984,6 @@
          </layout>
         </widget>
        </item>
-       <item row="1" column="0">
```

Comment:
I am going to climb onto my soapbox (and Vadi will probably try to push me off! :laughing: ) but if possible it is really helpful if you are editing things in a `QGridLayout` if you open it in the `Plain Text editor` afterwards and put the `<item row="'x'" column="'y'">` items back into row then column order. The Qt Designer plugin/utility tends to put them back in a random order after editing which makes the git diff much harder (almost impossible) to review for that file.

 Fortunately the plain text editor has the capability to fold (conceal lower - more indented - blocks of XML text) so you can click on the down/right triangles in the margin (which don't show up well in my current colour scheme) to change that and then select each item and cut and paste it to rearrange them into order - {though it gets a bit funny with maintaining the right order when pasting an item as the FIRST one and auto expands any that are pasted as the last one - so you have to be a bit tricksy in the sequence of operations. {Hint: if you are going do this - save a backup of the `.ui` file first to cover your back...!}

![image](https://user-images.githubusercontent.com/6163092/88408841-02125600-cdcc-11ea-922c-0ef4fdbcfa20.png)